### PR TITLE
Docs: Explain non-production / advanced install options in more detail

### LINKED ISSUE #49878: [ARM] Request to Specify ClickHouse Version during Installation
RPM packages only support modern ARM cores, such as Graviton 2, 3.

For older ARM cores, install ClickHouse as follows:
```
curl https://clickhouse.com/ | sh
```

_Originally posted by @alexey-milovidov in https://github.com/ClickHouse/ClickHouse/issues/49814#issuecomment-1546781571_

Dear alexey,

I hope this message finds you well. I recently installed ClickHouse 23.5.1 following the instructions provided, and I'm grateful for the guidance and support provided by the project.

I wanted to inquire about the possibility of specifying a particular version of ClickHouse during the installation process. Specifically, I am interested in installing version 23.2.4.12. I have reviewed the available documentation and configuration options, but I couldn't find a clear way to specify the desired version.

Could you please guide me on how to install a specific version of ClickHouse? It would be immensely helpful if you could provide instructions or point me to the relevant documentation or resources.

Thank you for your attention to this matter, and I appreciate the effort you have put into maintaining and improving ClickHouse. Your contributions have been invaluable to the community.

Best regards,
zenghuasheng

### PR DESCRIPTION
**Pull request by rschu1ze**
Fixes: #49878

### Changelog category (leave one):
- Documentation (changelog entry is not required)

### CODE REVIEW

**Comment by rschu1ze** [comment]
@DanRoscigno Would you like to read over my changes? Thanks!

File: docs/en/getting-started/install.md [hunk 1]
```diff
@@ -28,21 +28,23 @@ The quickest and easiest way to get up and running with ClickHouse is to create
...
 1. Run the following command to start the ClickHouse server:
+
     ```bash
     ./clickhouse server
     ```
 
-    The first time you run this script, the necessary files and folders are created in the current directory, then the server starts.
+   The first time you run this script, the necessary files and folders are created in the current directory, then the server starts.
 
 1. Open a new terminal and use the **clickhouse-client** to connect to your service:
```
Comment (DanRoscigno) [inline docs/en/getting-started/install.md:49]:
```suggestion
1. Open a new terminal and use the **./clickhouse client** to connect to your service:
```

Comment (DanRoscigno) [reply docs/en/getting-started/install.md:49]: unless they are on a system where install works they need to use `./clickhouse client`

File: docs/en/getting-started/install.md [hunk 2]
```diff
@@ -346,8 +350,31 @@ You’ll need to create data and metadata folders manually and `chown` them for
 
 On Gentoo, you can just use `emerge clickhouse` to install ClickHouse from sources.
 
-### From CI checks pre-built binaries
-ClickHouse binaries are built for each [commit](/docs/en/development/build.md#you-dont-have-to-build-clickhouse).
+### Install a CI-generated Binary
+
```
Comment (DanRoscigno) [inline docs/en/getting-started/install.md:354]: Are you going to remove https://github.com/ClickHouse/ClickHouse/blob/master/docs/en/development/build.md#you-dont-have-to-build-clickhouse-you-dont-have-to-build-clickhouse ? If this content needs to be in both places let me know and I will make it a snippet that can be single sourced.

**Review by DanRoscigno — CHANGES_REQUESTED** [review]

File: docs/en/getting-started/install.md (hunk 1 above)
Comment (rschu1ze) [reply docs/en/getting-started/install.md:49]: Oh, didn't know that (and I should know :-) )

**Comment by robot-ch-test-poll** [comment]
This is an automated comment for commit 8d6244ed991e30129a5f66aea85842a2c45d8520 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/50359/8d6244ed991e30129a5f66aea85842a2c45d8520/ci_running.html)
The overall status of the commit is 🔴 failure

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>Style Check</td><td>Runs a set of checks to keep the code style clean. If some of tests failed, see the related log from the report</td><td>🔴 failure</td></tr>
</table>

File: docs/en/getting-started/install.md (hunk 2 above)
Comment (rschu1ze) [reply docs/en/getting-started/install.md:354]: Good point. Guess I rather merge the existing instructions with the new ones. Let me make this more consistent.

**Review by DanRoscigno — APPROVED** [review]
LGTM

**Pull request merged**